migrate-demo-test.js

```diff
@@ -98,13 +98,44 @@ async function migrateDemoTest ({ web3, spinner, confirm, opts, migrationParams,
   const DAOToken = await avatar.methods.nativeToken().call()
   const Reputation = await avatar.methods.nativeReputation().call()
 
+  const DemoDAOToken = await new this.web3.eth.Contract(
```

Comment:
Why not just call it DAOToken ?
Why Demo ?

Reply:
Just to differentiate it to make sure it's not colliding (and won't in the future).